composer.json

```diff
@@ -27,5 +26,8 @@
         "allow-plugins": {
             "php-http/discovery": true
         }
+    },
+    "replace": {
+        "guzzlehttp/guzzle": "*"
```

Comment:
This would mean integration_onedrive replaces guzzle?

Reply:
I tried honestly different ways to fix this CI locally, but with `"guzzlehttp/guzzle"` in `replace` we need to manually put Guzzle into the CI - I can add it to this PR, but I'm not sure that it's correct.